src/apps/sommelier/positions.ts

```diff
@@ -0,0 +1,101 @@
+import { logger } from '../../log'
+import { getClient } from '../../runtime/client'
+import { getTokenId } from '../../runtime/getTokenId'
+import { toDecimalNumber } from '../../types/numbers'
+import {
+  PositionDefinition,
+  PositionsHook,
+  PricePerShareContext,
+  UnknownAppTokenError,
+} from '../../types/positions'
+import { cellarV0821Abi } from './abis/cellar'
+import { getSommStrategiesData } from './sommelierApi'
+
+const hook: PositionsHook = {
+  getInfo() {
+    return {
+      id: 'sommelier',
+      name: 'Sommelier',
+      description: 'Sommelier vaults',
+    }
+  },
+  async getPositionDefinitions({ networkId }) {
+    const client = getClient(networkId)
+    const cellars = await getSommStrategiesData(networkId)
+    const positionDefinitions: PositionDefinition[] = []
+
+    const results = await Promise.allSettled(
```

Comment:
Does it currently fail for some of the vaults?

Reply:
No that's good, it's better if it can still return what works.
I wonder if we should log an error instead of a warning, so we see something unexpected happened 🤔